alphabase/spectral_library/reader.py

```diff
@@ -170,7 +170,7 @@ def _get_fragment_intensity(self, lib_df: pd.DataFrame):
         ]
 
         # by default, all non-fragment columns are used to group the library
-        non_fragment_columns = list(set(lib_df.columns) - set(fragment_columns))
+        non_fragment_columns = sorted(list(set(lib_df.columns) - set(fragment_columns)))
```

Comment:
The line looks so innocent 😂

Reply:
took two hours of my life that I am going to reclaim once it's time 💀